front/app/modules/commercial/customizable_homepage_banner/admin/CTASettings/SettingRadioButtons.tsx

```diff
@@ -11,7 +11,7 @@ import CustomizedButtonSettings from './CustomizedButtonSettings';
 import { CLErrors } from 'typings';
 import styled from 'styled-components';
 
-const CustomizedButtonSettingsContainer = styled.div`
+const CustomizedButtonSettingsContainer = styled(CustomizedButtonSettings)`
```

Comment:
We'd call this StyledCustomizedButtonSettings in the code base, but no need to change 😁